web/components/notifications/notification-helpers.tsx

```diff
@@ -281,3 +271,41 @@ export function ParentNotificationHeader(props: {
     </Row>
   )
 }
+export function combineReactionNotifications(notifications: Notification[]) {
+  const groupedNotificationsBySourceType = groupBy(
+    notifications,
+    (n) => n.sourceType
```

Comment:
Not sure, but it might be simpler to just make this the combined key: `${n.sourceType},${n.sourceTitle}`. Then get the values of the object.

Reply:
that's much cleaner, thanks!